crucible-llvm/src/Lang/Crucible/LLVM/MemModel/MemLog.hs

```diff
@@ -204,47 +204,59 @@ possibleAllocInfo n (MemAllocs xs) = asum (map helper xs)
 
 
 -- | Generalized function for checking whether a memory region ID is allocated.
+--
+-- The first predicate indicates whether the region is allocated, the second
+-- indicates whether it was previously freed (useful for distinguishing
+-- use-after-free and double-free errors from other kinds of problems). The
+-- second predicate has reverse the polarity as that of the first - it is true
+-- just when the allocation was previously freed.
```

Comment:
I'm not convinced I love this polarity convention.  What's the reasoning here?

Reply:
Changed the polarity.